src/lib/tabs/tabs.component.ts

```diff
@@ -4,14 +4,14 @@ import { PlexTabComponent } from './tab.component';
 @Component({
     selector: 'plex-tabs',
     template: ` <section justify>
-                    <ul #container class="nav nav-tabs" [ngClass]="size">
-                        <li *ngFor="let tab of tabs" (click)="selectTab(tab, $event)" (auxclick)="closeTab(tab, $event)" class="nav-item nav-item-{{tab.color}}" [ngClass]="{'active': tab.active, 'icon': tab.icon && !tab.label}">
-                            <a class="nav-link" [ngClass]="{active: tab.active}" plexRipples onclick="return false">
-                                <plex-icon *ngIf="tab.icon" [name]="tab.icon" size="sm" [type]="tab.color"></plex-icon>
+                    <ul role="tablist" #container class="nav nav-tabs" [ngClass]="size" >
+                        <li role="presentation" *ngFor="let tab of tabs" (keydown.arrowright)="nextTab()" (keydown.arrowleft)="prevTab()" (keydown.enter)="selectTab(tab, $event)" (click)="selectTab(tab, $event)" (auxclick)="closeTab(tab, $event)" class="nav-item nav-item-{{tab.color}}" [ngClass]="{'active': tab.active, 'icon': tab.icon && !tab.label}">
+                        <a tabindex="{{ tab.active ? '0' : '-1' }}" role="tab" attr.aria-selected="{{ tab.active }}" attr.aria-label="{{ tab.label }}" id="{{ tab.label }}" class="nav-link" [ngClass]="{active: tab.active}" plexRipples onclick="return false">
```

Comment:
Recomiendo usar los bindings de angular:

```suggestion
                        <a [attr.tabindex]="tab.active ? '0' : '-1'" role="tab" [attr.aria-selected]="tab.active" [attr.aria-label]=" tab.label" [attr.id]="tab.label" class="nav-link" [ngClass]="{active: tab.active}" plexRipples onclick="return false">
```

Ojo con mezclar las cosas: Acá va un ejemplo : https://angular.io/guide/accessibility#accessibility-attributes

Reply:
Gracias @liquid36